src/examples.jl

```diff
@@ -1343,9 +1343,7 @@ function test_examples(pkgname::Symbol, idx::Int; debug = false, disp = true)
     map(exprs -> Base.eval(m, exprs), _examples[idx].exprs)
 
     plt = current()
-    if disp
-        gui(plt)
-    end
+    disp && Base.eval(m, :(gui(current())))
```

Comment:
```suggestion
    disp && Base.eval(m, :(gui(plt)))
```

Reply:
That won't work, since plt is not assigned in the same scope.